SQRLDotNetClientUI/ViewModels/AuthenticationViewModel.cs

```diff
@@ -141,6 +149,14 @@ private void Init()
             });
 
             CheckForQuickPass();
+
+            CheckForUpdate();
+
+        }
+
+        private async void CheckForUpdate()
+        {
+            this.NewUpdateAvailable = await GitHubApi.GitHubHelper.CheckForUpdates();
```

Comment:
If we await this here, aren't we then back to square one and everything is in fact synchronous again?
Or is the surrounding `CheckForUpdate()` called async and so it doesn't matter?

Reply:
No await is smart it continues since nothing depends on the result

Reply:
Await only waits if it needs to in this case nothing depends (down the logic) on its result so it continues

Reply:
Await only waits if it needs to in this case nothing depends (down the logic) on its result so it continues